web-server/v0.4/src/components/PageHeader/index.js

```diff
@@ -204,6 +205,19 @@ export default class PageHeader extends PureComponent {
     if (tabActiveKey !== undefined) {
       activeKeyProps.activeKey = tabActiveKey;
     }
+    let titleHeader = '';
+    if (selectedControllers !== undefined) {
+      titleHeader = selectedControllers.map(i => (
+        <Tag color="blue" style={{ fontSize: '105%' }}>
+          {i}
+        </Tag>
+      ));
+      titleHeader = <p>Selected Controllers : {titleHeader} </p>;
```

Comment:
Can we consider referencing the titleHeader as `titleHeader = <div><h1 className={styles.title}>Selected Controllers</h1>{titleHeader}</div>` for visual consistency with title headers in other page components?